src/main.dart

```diff
@@ -1,8 +1,129 @@
-// TODO: Run task list app
-void main(List<String> args) {
-  print('Hello World');
+import 'dart:io';
+import 'task.dart';
+import 'taskList.dart';
+
+//you can make it global so you can access it from anywhere
+List<Task> _tasks = [];
+TaskList tasks = TaskList([]);
+
+void main(List<String> arg) {
+//Listing Choices
+  void showTasks() {
+    print("This is your Task List app");
+    print("**************************");
+    print("What Do You Need?");
+    print("1.Add Task");
+    print("2.List Task");
+    print("3.Update A Task");
+    print("4.Delete A Task");
+    print("5.Exit");
+  }
+
+//Making Choice
+  int choice = 0;
+  while (choice != 5) {
+    showTasks();
+    bool validChoice = false;
+    while (!validChoice) {
+      try {
+        stdout.write("Your Choice?");
+        choice = int.parse(stdin.readLineSync()!);
+        validChoice = true;
+      } catch (e) {
+        print("Enter A Number:");
+      }
+    }
+//Operating Choice
+    switch (choice) {
+      case 1:
+        addTask();
+        break;
+      case 2:
+        printTasks();
+        break;
+      case 3:
+        choose();
+        break;
+      case 4:
+        tasks.remove();
+        break;
+      default:
+        print("Invalid Choice");
+        break;
+    }
+  }
+}
+
+//Choose Between Title && Description Update
+
+void choose() {
+  int choice = 0;
+  try {
+    stdout.write("To Change Title:1 / To Change Description:2 \n Your Choice?");
+    choice = int.parse(stdin.readLineSync()!);
+    //validChoice = true;
+  } catch (e) {
+    print("Enter A Number:");
+  }
+  switch (choice) {
+    case 1:
+      updateTitle(_tasks);
+      break;
+    case 2:
+      updateDescription(_tasks);
```

Comment:
unnecessary variable
```suggestion
      updateDescription();
```